pkg/search/postgres/common.go

```diff
@@ -1077,6 +1077,7 @@ func RunCursorQueryForSchemaFn[T any, PT pgutils.Unmarshaler[T]](ctx context.Con
 		return errors.Wrap(err, "creating cursor")
 	}
 
+	const cursorBatchSize = 1000
```

Comment:
nit:  I probably wouldn't have buried the const here but at the top.  But that could simply be preference so do with what you will.

Reply:
top of the file or function?

Reply:
I was thinking file

Reply:
OK, I moved it to the function as it's only used there. Having it in the file will make it package scope but I think it wont hurt us.